less/variables.less

```diff
@@ -269,6 +269,14 @@
 @spacing-3-tablet: 72px;
 @spacing-3-desktop: 96px;
 
+@spacing-lg-mobile: 80px;
+@spacing-lg-tablet: 120px;
+@spacing-lg-desktop: 160px;
+
+@spacing-xl-mobile: 112px;
+@spacing-xl-tablet: 168px;
+@spacing-xl-desktop: 224px;
```

Comment:
Why would the naming convention change?  I deliberately stayed away from using lg, xl etc because they are the names of our grid sizes, so using them for spacing is a little confusing.  The smaller spacing classes already use numbers.

Reply:
Because I didn´t follow the system logical increase, I though that changing the naming made sense.